src/code/actions.js

```diff
@@ -409,7 +409,8 @@ function _acceptEggInBasket(eggDrakeIndex, basketIndex) {
   return {
     type: actionTypes.EGG_ACCEPTED,
     eggDrakeIndex,
-    basketIndex
+    basketIndex,
+    meta: {sound: 'receiveCoin'}
```

Comment:
Now playing a sound when an egg is accepted, until some kind of visual cue can be shown.